CONTRIBUTING.md

```diff
@@ -124,18 +124,27 @@ This script generates:
    - Accurate description and usage examples
    - Correct icon path (usually `../../../../.icons/your-icon.svg`)
    - Proper tags that describe your module
-3. **Create at least one `.tftest.hcl`** to test your module with `terraform test`
+3. **Create tests for your module:**
+   - **New modules**: Create `.tftest.hcl` files and test with `terraform test`
```

Comment:
Let's also include unit testing for new modules, as we can not test the business logic of scripts with Terraform-only tests.